src/itemviewformaction.cpp

```diff
@@ -245,17 +245,7 @@ bool ItemViewFormAction::process_operation(Operation op,
 	case OP_OPENINBROWSER:
 	case OP_OPENBROWSER_AND_MARK: {
 		LOG(Level::INFO, "ItemViewFormAction::process_operation: starting browser");
-		v->set_status(_("Starting browser..."));
```

Comment:
I think this message should be kept, because it's useful to people with weird `browser` settings like `browser "firefox"`: if Firefox is not yet running, `OP_OPENINBROWSER` will start it *and block Newsboat* until Firefox is closed. This message provides some indication to the user as to what's going on. Without it, the user might think that Newsboat simply hung.

I notice that other dialogs don't display such a message, and I think that's a bug that should be fixed. I can submit a PR for that once this one is merged, or you can fix it in this PR — I'm okay with both.

Reply:
I believe the message (`"Starting browser..."`) is not visible anyway.
It is displayed in the bottom of a Newsboat FormAction but the `open_in_browser()` call will directly change the terminal mode, hiding the Newsboat FormAction (`Stfl::reset()`).
It is only restored when the browser process has finished, so I don't think the message is useful.

Reply:
Hmm, yeah, you're right. Disregard this comment then.